.changeset/move_autopilots_into_autopilot_state.md

```diff
@@ -0,0 +1,9 @@
+---
+default: minor
```

Comment:
Should bump this to `major`

Reply:
You can install `knope` locally and run `knope document-change` to generate a blank one. 

Either: `cargo install knope` if you have Rust installed or download the binary.